src/anet.c

```diff
@@ -569,11 +580,11 @@ int anetUnixAccept(char *err, int s) {
     return fd;
 }
 
-int anetFdToString(int fd, char *ip, size_t ip_len, int *port, int fd_to_str_type) {
+int anetFdToString(int fd, char *ip, size_t ip_len, int *port, int addr_type) {
     struct sockaddr_storage sa;
     socklen_t salen = sizeof(sa);
 
-    if (fd_to_str_type == FD_TO_PEER_NAME) {
+    if (addr_type == ADDR_PEER_NAME) {
```

Comment:
Because RDMA also needs to get local/remote info, but it's not an `ANET_PEER_NAME`, so I renamed this into `ADDR_PEER_NAME`. And yes, using `ADDR_PEER_NAME`(defined in connection.h) in anet.c is not good enough.

As you suggested, what about `bool remote`?

Reply:
bool (int) sounds like a good compromise.